### PR TITLE
Романов - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by xnd-r**

### CODE REVIEW

**Comment by xnd-r** [comment]
@katerina394 @VladimirNechesanov please review

**Comment by katerina394** [comment]
Name of class DepoCalc not selfexplanatory, rename, please

File: code/depo_calc/model/depo_calc.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        assert all([s0 > 0,
+                    t > 0,
+                    r > 0,
+                    r < 1,
+                    is_add_to_depo in [True, False],
+                    capitalization_freq in [1, 2, 3, 4, 6, 12],
+                    replenishment_freq in [-1, 1, 2, 3, 4, 6, 12],
+                    t * 12 >= capitalization_freq,
+                    capitalization_freq > replenishment_freq
+                    ])
```
Comment (katerina394) [inline code/depo_calc/model/depo_calc.py:20]: Unreasonable count of newlines

**Comment by katerina394** [comment]
👍

**Comment by VladimirNechesanov** [comment]
👍

**Pull request merged**